packages/asset-service/src/service/AssetService.ts

```diff
@@ -114,13 +119,24 @@ export class AssetService {
     return result
   }
 
-  async description({ asset }: { asset: Asset }): Promise<string> {
+  async description({ asset }: { asset: Asset }): Promise<DescriptionData> {
     const descriptions: Record<string, string> = assetsDescriptions
 
-    // Return overriden asset description if it exists
-    if (descriptions[asset.caip19]) return descriptions[asset.caip19]
+    // Return overriden asset description if it exists and add isTrusted for description links
+    if (descriptions[asset.caip19]) {
+      return {
+        description: descriptions[asset.caip19],
+        isTrusted: true
+      }
+    }
+
+    if (asset.dataSource !== AssetDataSource.CoinGecko) {
```

Comment:
we *only* want to trust the `descriptions.json` file, everything else is still untrusted

Reply:
Totally agree. The only description object that is returned with `isTrusted` as true is when we find a description for a specific caip19 on line 126.